client/app/components/proptypes.js

```diff
@@ -65,6 +65,7 @@ export const Query = PropTypes.shape({
 
 export const AlertOptions = PropTypes.shape({
   column: PropTypes.string,
+  selector: PropTypes.oneOf(["max", "min", "first"]),
```

Comment:
Let's change the order to `["first", "min", "max"]`

Reply:
Done